letsdrill.rb

```diff
@@ -1,24 +1,34 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
+  unless integer.class == Fixnum && integer.between?(0, 100)
```

Comment:
:+1: nice use of `.between?`
`integer.is_a? Fixnum` is another option here.

I love the checks on integer here. It's important to check assumptions at the door. Note: You'll see 'validators' & 'normalizers' used in cmm api's.